ts/Core/Series/DataLabel.ts

```diff
@@ -608,14 +608,36 @@ namespace DataLabel {
                                 point.contrastColor = renderer.getContrast(
                                     (point.color || series.color) as any
                                 );
+
+                                const background = chart.chartBackground,
+                                    plotBackground = chart.plotBackground;
+
+                                let elseColor = String(Palette.neutralColor100);
```

Comment:
I think using the JS `String()` function is not necessary here because we're dealing with a TS error.

Better (but not good) solution would be to cast it ti String like this:
```ts
let elseColor = Palette.neutralColor100 as String;
```

The best solution would be to tell TypeScript that this is actually a `string` type variable since the very beginning of its declaration - TS definitions are fine now and we don't use any JS code.